### PR TITLE
Feature: Grid with 2 themes

### PR DESCRIPTION
**Pull request by alejandroclose**
- Grid component switched from class component to functional component.
- Added theme to grid (2): `grid-icons-red` & `grid-images-red`.
- Added type to cards (2): `icons` & `images`.

### CODE REVIEW

File: src/pages/home/grid/grid.jsx [hunk 1]
```diff
@@ -1,62 +1,39 @@
...
-//Mocks
-import { MOCK_GRID_ELEMENTS } from 'mocks/mockGridElements';
+const Grid = (props) => {
+  const [showCards, setShotCards] = useState(4);
+  return (
+    <GridWrapper >
+      <GridTitle>
+        <h2>{props.title}</h2>
+      </GridTitle>
+      <GridCards className={props.theme}>
```
Comment (mad-js-dev) [inline src/pages/home/grid/grid.jsx]: Esto se debería llamar GridItems

File: src/pages/home/grid/grid.style.js [hunk 2]
```diff
@@ -2,21 +2,64 @@
...
+  color: #EA3840;
   display: flex;
   flex-direction: column;
   flex-wrap: wrap;
   align-items: center;
   text-align: center;
   justify-content: center;
   background-color: #f9f6ef;
+
+&.grid-icons-red{
```
Comment (mad-js-dev) [inline src/pages/home/grid/grid.style.js]: Esto deberia llamarse grid a secas.

File: src/pages/home/grid/grid.style.js [hunk 3]
```diff
@@ -2,21 +2,64 @@
...
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  align-items: center;
+  text-align: center;
+  justify-content: space-evenly;
+  background-color: #f9f6ef;
+}
+
+&.grid-images-red{
```
Comment (mad-js-dev) [inline src/pages/home/grid/grid.style.js]: Y esto grid-fluid

File: src/pages/home/grid/grid.style.js [hunk 4]
```diff
@@ -2,21 +2,64 @@
 import styled from 'styled-components';
```
Comment (mad-js-dev) [inline src/pages/home/grid/grid.style.js]: este componente deberia estar en shared

**Review by mad-js-dev — COMMENTED** [review]
Please review comments

File: src/pages/home/grid/grid.jsx (hunk 1 above)
Comment (alejandroclose) [reply src/pages/home/grid/grid.jsx]: Hecho! He cambiado además de elements a items la prop donde se pasa el mock.

File: src/pages/home/grid/grid.style.js (hunk 2 above)
Comment (alejandroclose) [reply src/pages/home/grid/grid.style.js]: Canviado a grid

File: src/pages/home/grid/grid.style.js (hunk 3 above)
Comment (alejandroclose) [reply src/pages/home/grid/grid.style.js]: canviado a grid-fluid

**Comment by alejandroclose** [comment]
@mad-js-dev Changes done! :)

File: src/pages/home/home.jsx [hunk 5]
```diff
@@ -1,21 +1,42 @@
 // Dependencies
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 
 // Components
 import Hero from './hero/hero';
 import { Carousel } from 'shared';
-import Grid from './grid/grid';
+import Grid from '../../shared/components/grid/grid';
```
Comment (mad-js-dev) [inline src/pages/home/home.jsx]: Why grid/grid? Shouldnt be just /grid?
Why are you calling shared with the ../../ line 6 is already calling shared, add it there.

File: src/shared/components/grid/grid_OLD.jsx [hunk 6]
```diff
@@ -2,41 +2,29 @@
 import React, { Component } from 'react';
```
Comment (mad-js-dev) [inline src/shared/components/grid/grid_OLD.jsx]: Not reallu sure why are we leaving the old version of the component. Is there a reason?

File: src/pages/home/home.jsx (hunk 5 above)
Comment (alejandroclose) [reply src/pages/home/home.jsx]: You're right!! Updated!

File: src/shared/components/grid/grid_OLD.jsx (hunk 6 above)
Comment (alejandroclose) [reply src/shared/components/grid/grid_OLD.jsx]: No reason, sorry!

**Pull request merged**